src/requireDatabaseInstance.ts

```diff
@@ -0,0 +1,34 @@
+import * as clc from "cli-color";
+import { getDefaultDatabaseInstance } from "./getDefaultDatabaseInstance";
+import * as utils from "./utils";
+
+/**
+ * Ensures that the supplied options have an instance set. If not, tries to fetch the default instance.
+ * @param options command options
+ */
+export async function requireDatabaseInstance(options: any) {
+  if (options.instance) {
+    return Promise.resolve();
```

Comment:
In an `async` function, you do not need to manually return resolved promises. Applies throughout.

```suggestion
    return;
```